compile.py

```diff
@@ -40,140 +40,183 @@ def create_view(dataset_name, view_name, project, viewSQL):
     DATASET = ""
     PROJECT = ""
 
     parser = argparse.ArgumentParser()
-    
-    parser.add_argument("-m", "--metric_file", metavar="METRIC", action="store", help="the metric filename to process")
-    parser.add_argument("-o", "--output_file", metavar="OUTPUTFILE", action="store")
-    parser.add_argument("-v", "--create_view", action="store_true", help="create a view from the metric")
-    parser.add_argument("-a", "--all_metrics", action="store_true", help="deploy all views")
-    parser.add_argument("-d", "--dataset", metavar="DATASET", action="store", help="deploy views to this dataset")
-    parser.add_argument("-p", "--project", metavar="PROJECT", action="store", help="deploy views to a dataset in this project")
 
-    args = parser.parse_args()  
+    parser.add_argument(
+        "-m",
+        "--metric_file",
+        metavar="METRIC",
+        action="store",
+        help="the metric filename to process",
+    )
+    parser.add_argument("-o", "--output_file", metavar="OUTPUTFILE", action="store")
+    parser.add_argument(
+        "-v", "--create_view", action="store_true", help="create a view from the metric"
+    )
+    parser.add_argument(
+        "-a", "--all_metrics", action="store_true", help="deploy all views"
+    )
+    parser.add_argument(
+        "-d",
+        "--dataset",
+        metavar="DATASET",
+        action="store",
+        help="deploy views to this dataset",
+    )
+    parser.add_argument(
+        "-p",
+        "--project",
+        metavar="PROJECT",
+        action="store",
+        help="deploy views to a dataset in this project",
+    )
+
+    args = parser.parse_args()
 
     if not (args.all_metrics or args.metric_file):
         parser.print_usage()
         sys.exit(1)
-        
-    if (not args.all_metrics):
+
+    if not args.all_metrics:
         with indent(4, quote="* "):
             puts(colored.green("Metric file is ['{}']".format(args.metric_file)))
-            if (args.output_file):
+            if args.output_file:
                 puts(colored.green("Output file is ['{}']".format(args.output_file)))
-            puts(colored.blue("Rendering ['{}']".format(args.metric_file.replace(".j2", ""))))
+            puts(
+                colored.blue(
+                    "Rendering ['{}']".format(args.metric_file.replace(".j2", ""))
+                )
+            )
 
         finalSQL = render_template(args.metric_file, args.output_file)
 
-        if (args.create_view):
+        if args.create_view:
             view_sql = finalSQL
             view_name = args.metric_file.split(".")[0]
             create_view(args.dataset, view_name, args.project, view_sql)
 
-        if (args.output_file):
+        if args.output_file:
             print("...SQL File built at ./build/{}".format(args.output_file))
     else:
         deploy_all(args.dataset)
 
     with indent(4, quote="* "):
         puts(colored.yellow("Finished!"))
-      
+
+
 if __name__ == "__main__":
-   main()
+    main()
```

Comment:
Is the indentation wrong here?

Reply:
No, 4 spaces. Looks like you had 3 before.